rust/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "imtl-rs"
+version = "0.1.0"
+edition = "2021"
```

Comment:
nit: why it's not 2023

Reply:
This is the latest released edition, the next edition is 2024🙂

Reply:
ok, 3x